src/share/doc/apidiff.md

```diff
@@ -22,20 +22,17 @@
 # questions.
 #
 
-title: 'APIDIFF(1) 0.1 | CodeTools'
-date: 2021
+title: 'APIDIFF(1) 1.0 | CodeTools'
+date: 2024
```

Comment:
Suggest you update the copyright year on line 3 too.

Reply:
Thank you; will do.